frontend/src/pages/Admin/Components/Testimonial/ManageTestimonial/ManageTestimonial.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from "react";
+import style from "./manage-testimonial.module.scss";
+import { SimpleToast } from "../../../../../components/util/Toast/Toast";
+import Loader from "../../../../../components/util/Loader";
+import { deleteTestimonial, getTestimonials } from "../../../../../service/Testimonial";
+
+export function ManageTestimonial() {
+  const [testimonials, setTestimonials] = useState([]);
+  const [images, setImages] = useState([]);
+  const [toast, setToast] = useState({
+    toastStatus: false,
+    toastType: "",
+    toastMessage: "",
+  });
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const getdata = async () => {
+    setIsLoaded(true);
+    await getTestimonials(setTestimonials,setToast);
+    setIsLoaded(false);
+  };
+
+  const handleDelete = async (id) => {
+    setIsLoaded(true);
+    await deleteTestimonial(id, setToast, toast);
+    await getdata();
+    setIsLoaded(false);
+  };
+  const handleCloseToast = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    setToast({ ...toast, toastStatus: false });
+  };
+
+  useEffect(() => {
+    getdata();
+  }, []);
+
+  return (
+    <div>
+      <h1 className={style["head"]}>Manage Testimonials</h1>
+      {isLoaded ? (
+        <div className={style["data-loader"]}><Loader /></div>
+      ) : (
+        <div className={style["manage-testimonials"]}>
+          {testimonials?.map((testimonial, index) => (
+            <div className={style["crd"]} key={index}>
+              <div style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
+                <img
+                  src={
+                    (testimonial.image) ||
+                    "./images/defaultUser.png"
+                  }
```

Comment:
please try to avoid inline styling and read from scss file instead

Reply:
Done Updated.